### PR TITLE
#4563 Updated French translations from spreadsheet

### LINKED ISSUE #4563: Corrections to French translations 🇫🇷
## Is your feature request related to a problem? Please describe.

While doing a demo in French, Pauline noticed some typos in the French translations for mSupply mobile.
These have been updated in the spreadsheet here:
https://docs.google.com/spreadsheets/d/1yNEwvHbq__1GWCw7Bul-dnqnnPINVa2CbNcCYg-zOFE/edit#gid=1174361042

## Describe the solution you'd like

We should import these for the next mobile release!

## Implementation

Use the import tool to update the french translations in mobile, then test to make sure it's all looking ok 👀  (sometimes the prompts get a bit long and things get cut off)

## Describe alternatives you've considered

Leaving the typos and looking unprofessional? 🧐 

## Additional context
N/A

### PR DESCRIPTION
**Pull request by katherine-sussol**
Fixes #4563

## Change summary
Updates as kindly provided by Pauline!

## Testing

Steps to reproduce or otherwise test the changes of this PR:

- [ ] Check the buttons for 'use requested quantities' and 'use suggested quantities' (Customer Requisitions)
- [ ] Check the title for 'suggested_equals' pageInfo (Customer Requisitions)
image
- [ ] Check the 'current_stock' label (Menu Page and column heading for supplier requisitions)
image
image

### Related areas to think about

If there are any general areas of the codebase your changes might have side affects on, mention them here

### CODE REVIEW

File: src/localization/buttonStrings.json [hunk 1]
```diff
@@ -68,8 +68,8 @@
     "realm_explorer": "Explorateur Realm",
     "save_changes": "Sauvegarder les modifications",
     "settings": "RÉGLAGES",
-    "use_requested_quantities": "Utiliser qté requises",
-    "use_suggested_quantities": "Utiliser qté suggérées",
+    "use_requested_quantities": "Utiliser quantité requise",
```
Comment (katherine-sussol) [inline src/localization/buttonStrings.json:71]: This one looks to be cut off due to the length (whereas Utiliser qté requises just seems to fit)

Comment (katherine-sussol) [reply src/localization/buttonStrings.json]: Fixed by abbreviating (and showing abbreviation) in [5835e67](https://github.com/openmsupply/mobile/pull/4648/commits/5835e67eac320503c79269ae97d6bf50402bd555) and [221735c](https://github.com/openmsupply/mobile/pull/4648/commits/221735ca69eed884f6b340cffa622b0b0b866d06)

**Comment by katherine-sussol** [comment]
screen

With [9eab555](https://github.com/openmsupply/mobile/pull/4648/commits/9eab555930f07b8dcc659984079fb7893db9f3b3) have tried to expand the top left panel (buttons don't need too much space on any of the screens) to accommodate the lengthier labels a bit better.

**Review by clemens-msupply — APPROVED** [review]
Haven't tested the flex change but looks good

**Pull request merged**